src/react/projects/spark-react/src/SprkLink/SprkLink.js

```diff
@@ -10,21 +10,36 @@ const SprkLink = props => {
     additionalClasses,
     idString,
     analyticsString,
+    href,
+    onClick,
     ...other
   } = props;
   const TagName = element || 'a';
```

Comment:
What is `element`? Are `SprkLink`s always `<a>`?

Reply:
Talked to Rob - we need to expose the option for an element to be passed so they can pass a router link  for internal pages. If the <a> is passed it will automatically reload the page. I'll add "element" to the docs.